### PR TITLE
skills consistency feats

### PR DESCRIPTION
**Pull request by rosebrallier**
Changed defunct skills to their new equivalent
Changed formatting conventions for skills
Edited some feats' skills-relevant descriptions for clarity or style
Pursuant to Issue #372

### CODE REVIEW

File: CharacterCreation/03_Feats.txt [hunk 1]
```diff
@@ -305,11 +305,11 @@ damages an area, Last Shot Counts does not take effect.
...
 
 Prerequisites:
-*  Pilot or 40 skill in piloting
+*  Pilot or 40 skill in Vehicles - Flight OR Space
 
-	You get a +20% to pilot skill checks in airborne or spaceborne vehicles, and
-an additional +20% to unconventional maneuvers in such as well, but you and
-your vehicle take double damage for the next three turns after using these
+	You gain a +20% to Vehicles checks in either airborne or spaceborne vehicles
+(choose one), and an additional +20% to unconventional maneuvers in such as well,
```
Comment (Turtlelord26) [inline CharacterCreation/03_Feats.txt]: I wouldn't mind giving both with the one instance of the feat (would also consider explicitly stating that you can take the feat twice to be able to apply it to both vehicle types). I don't think I like getting one and forever locking out the other.

File: CharacterCreation/03_Feats.txt [hunk 2]
```diff
@@ -124,8 +124,8 @@ Prerequisites:
 *  None
 
 	Make a stealth check to hide your chosen pistol, shotgun, or SMG on your 
-person. To succeed, you need a minimum score of 20. Roll determines DC for 
-enemy to actively search and find your hidden weapon.
+person. To succeed, you must pass a DC 19 check. Total roll determines DC for an
```
Comment (Turtlelord26) [inline CharacterCreation/03_Feats.txt:114]: Aren't skill checks success on tie? DC not-divisible-by-5 is very unusual.

**Review by Turtlelord26 — CHANGES_REQUESTED** [review]
👍

File: CharacterCreation/03_Feats.txt (hunk 2 above)
Comment (rosebrallier) [reply CharacterCreation/03_Feats.txt:114]: To quote basic rules: "Anytime you 
need to Pass a Skill check, there will be a DC (Difficulty Check) you have to 
exceed in order to successfully pass the check."
I'll happily change it back to 20.

Comment (Turtlelord26) [reply CharacterCreation/03_Feats.txt:114]: Nope, you're correct. DC 19 is the equivalent and more style-consistent phrasing. Disregard.

**Review by bleehu — APPROVED** [review]
I too would like to see some small changes to leaf on the wind that Turtle suggested, but I see those changes as non-blocking. Thanks a bunch for cleaning up these details! You're really helping to sanitize the confusion out of our rules! Shipit!

**Comment by rosebrallier** [comment]
I swapped Leaf back to giving both space and air. After thinking about it a little, if you /want/ to use Leaf in a vehicle you don't have a ton of skill in, I figure the risk balances out the benefit.

**Review by Turtlelord26 — APPROVED** [review]

**Comment by Turtlelord26** [comment]
There is also the possible confusion and rules lookups from the particular case of flying a spaceship through atmosphere that that change nullifies 👍

**Pull request merged**